bears-router.js

```diff
@@ -0,0 +1,84 @@
+const router = require('express').Router();
+
+const knex = require('knex');
+const knexConfig = {
+    client: 'sqlite3',
+    useNullAsDefault: true,
+    connection: {
+        filename: './data/lambda.sqlite3',
+    },
+    // debug: true,
+};
+const db = knex(knexConfig);
```

Comment:
it's great that you're still implementing express Routers as it cleans up your app quite a bit, but the next concern you should be recognizing is that you're being forced to then declare this knexConfig object in both router files--not very DRY. To handle that, you can abstract this object and the importing of knex into a dbConfig file (my preference and what you'll probably see elsewhere, but you can name it whatever you want) and export it, then import that reference in both router files instead